sandbox.js

```diff
@@ -1,7 +1,13 @@
 var tests = [
-  {text:"Write code that will console.log out 5 and 6", logs: ['5', '6'], vars: []},
-  {text:"Write code that will set x = 5, y = 6, and z = 9", logs: [], vars: [{name: "x", val: 5}, {name: "y", val: 6}, {name: "z", val: 9}]}
-]
+  {text:"Write code that will console.log out 5 and 6", logs: ['5', '6'], vars: [], functs: []},
+  {text:"Write code that will set x = 5, y = 6, and z = 9", logs: [], vars: [{name: "x", val: 5}, {name: "y", val: 6}, {name: "z", val: 9}], functs: []},
+  {text:"Correct function <b>convertFtoC</b> so that it correctly takes in a degrees in fahrenheit, returns in degrees celcius, then write a function <b>convertCtoF</b> to reverse the conversion",
+   logs: [], vars: [], functs: [{name: "convertFtoC", 
+                                tests: [{input: "32", output: "0"},{input: "82", output: "27.77777777777778"}]},
+                                {name: "convertCtoF", 
+                                tests: [{input: "0", output: "32"},{input: "27.77777777777778", output: "82"}]}
+                              ]}
+];
```

Comment:
Inputs should probably be an array because we want to be able to test functions with multiple inputs.

The other way I was thinking we could do this if we were feeling extra would be to allow you to pass in a working copy of the function and some inputs, and then we could test if the outputs matched. This test based method though feels a lot more reasonable especially once we make this into a more usable form.